src/main/java/org/stellar/sdk/requests/TooManyRequestsException.java

```diff
@@ -5,17 +5,7 @@
  * @see <a href="https://developers.stellar.org/api/introduction/rate-limiting/" target="_blank">Rate Limiting</a>
  */
 public class TooManyRequestsException extends RuntimeException {
-  private int retryAfter;
-
-  public TooManyRequestsException(int retryAfter) {
+  public TooManyRequestsException() {
```

Comment:
retyAfter variable can still be used by the outside developer.
You should either give some reasonable default, or make retryAfter a nullable Integer

Reply:
I'm closing this PR. There's already another PR opened that just need to merge